### PR TITLE
Update bestpractices.md

### PR DESCRIPTION
**Pull request by csvasque**
test

### CODE REVIEW

File: user-guide/docs/curating/bestpractices.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+Go Long horns! The DDR accepts engineering datasets generated through simulation, hybrid simulation, experimental, and field research methods regarding the impacts of wind, earthquake, and storm surge hazards, as well as debris management, fire, and blast explosions. We also accept social and behavioral sciences (SBE) data encompassing the study of the human dimensions of hazards and disasters. As the field and the expertise of the community evolves we have expanded our focus to include datasets related to COVID-19. Data reports, publications of Jupyter notebooks, code, scripts, lectures, and learning materials are also accepted.
```
Comment (wesleyboar) [inline user-guide/docs/curating/bestpractices.md]: Long horns. really? Okay.

**Review by wesleyboar — CHANGES_REQUESTED** [review]
Woah

File: user-guide/docs/curating/bestpractices.md (hunk 1 above)
Comment (wesleyboar) [reply user-guide/docs/curating/bestpractices.md]:
```suggestion
Go mustangs! The DDR accepts engineering datasets generated through simulation, hybrid simulation, experimental, and field research methods regarding the impacts of wind, earthquake, and storm surge hazards, as well as debris management, fire, and blast explosions. We also accept social and behavioral sciences (SBE) data encompassing the study of the human dimensions of hazards and disasters. As the field and the expertise of the community evolves we have expanded our focus to include datasets related to COVID-19. Data reports, publications of Jupyter notebooks, code, scripts, lectures, and learning materials are also accepted. 
```

Comment (csvasque) [reply user-guide/docs/curating/bestpractices.md]: I changed it

**Review by wesleyboar — APPROVED** [review]
Amazing.

**Comment by wesleyboar** [comment]
This was a test/learning PR. I close it now.

@csvasque, because your work was on `main` "branch" in https://github.com/csvasque/DS-User-Guide, when you do contribute _for real_, you should undo those changes. Making changes on a new "branch", e.g. like you did with [`patch-1`](https://github.com/csvasque/DS-User-Guide/tree/patch-1) [will avoid needing to undo changes that you retract or an admin declines].

**Comment by wesleyboar** [comment]
[…] will avoid needing to undo changes that you retract or [an admin declines].

**Pull request closed without merging**